tests/conftest.py

```diff
@@ -111,12 +96,28 @@ def mock_run_bgp_command(vtysh_cmd, bgp_namespace):
             return mock_frr_data
         return ""
 
-    def mock_run_bgp_ipv6_err_command(vtysh_cmd, bgp_namespace):
-        return "% Unknown command: show ipv6 route garbage"
+    def mock_run_show_ip_route_commands(request):
+        if request.param == 'ipv6_route_err':
+            return show_ip_route_common.show_ipv6_route_err_expected_output
+        elif request.param == 'ip_route':
+            return show_ip_route_common.show_ip_route_expected_output
+        elif request.param == 'ip_specific_route':
+            return show_ip_route_common.show_specific_ip_route_expected_output 
+        elif request.param == 'ip_special_route':
+            return show_ip_route_common.show_special_ip_route_expected_output
+        elif request.param == 'ipv6_route':
+            return show_ip_route_common.show_ipv6_route_expected_output
+        elif request.param == 'ipv6_specific_route':
+            return show_ip_route_common.show_ipv6_route_single_json_expected_output
+        else:
+            return ""
```

Comment:
Thoughts on having this as a dictionary and returning `show_ip_route_common.expected_output.get("ipv6_route_err", "")`?

Reply:
@daall Interesting idea!
Let me try it out.

Reply:
@daall Unfortunately I see cases where it is not a 1 to 1 mapping and it is test dependent...  So I will stick with the current method for now...